### ISSUE TITLE
Badge icons and NuGet

### ISSUE THREAD

**Issue by clairernovotny** [dotnet-foundation/project-maturity-model#6]
In addition to displaying the badges on a project's readme, I think it'd be useful to the ecosystem if the badges could be displayed in the NuGet UX (website, clients). I'd suspect that most developers aren't going to a GitHub site to determine quality. Having those badges visible where they look for packages could be a huge benefit and help users make a decision based on a project's maturity level.

**Comment by richlander** [comment]
Indeed. The proposal touches on this, and I've had initial conversations with the NuGet team on this. Let's assume that this will happen.

**Comment by anangaur** [comment]
This should just get enabled as part of ReadMe integration. i.e. when we allow developers to put a the markdown file in the nupkg and is displayed as extended description. The badges come automatically.

**Comment by clairernovotny** [comment]
@anangaur I would want to see this on the top-level search results, not buried in the readme.

**Comment by anangaur** [comment]
Got it. We are discussing about quality indicators and it does fit in that. Will keep you posted as we make progress.

**Comment by richlander** [comment]
Cool. I'll also setup some time with you @anangaur on what I see as the best next steps for NuGet to integrate into this.

**Comment by terrajobst** [comment]
Oren's ask obviously would mean integration work in the client UI as well. I could even see this going to be a step further and in conjunction with the trust features NuGet has built. For instance, a project might want to state "I'm a level X library, so please block me from accidentally depending on a library that isn't at least level X as well". Same goes for app authors in conservative environments that require the highest maturity level for dependencies. I suppose both will need a way to deal with exceptions tho.
